showOutput3.js

```diff
@@ -311,22 +248,21 @@ const longFormKeycodes = {
     KC_QUESTION: 'KC_QUES',
 
     // Unicode Support
-    UNICODE_MODE_FORWARD: 'UC_MOD',
-    UNICODE_MODE_REVERSE: 'UC_RMOD',
-    UNICODE_MODE_OSX: 'UC_M_OS',
-    UNICODE_MODE_LNX: 'UC_M_LN',
-    UNICODE_MODE_WIN: 'UC_M_WI',
-    UNICODE_MODE_BSD: 'UC_M_BS',
-    UNICODE_MODE_WINC: 'UC_M_WC'
+    UNICODE_MODE_FORWARD: 'UC_NEXT',
+    UNICODE_MODE_REVERSE: 'UC_PREV',
+    UNICODE_MODE_OSX: 'UC_MAC',
+    UNICODE_MODE_LNX: 'UC_LINX',
+    UNICODE_MODE_WIN: 'UC_WIN',
+    UNICODE_MODE_BSD: 'UC_BSD',
+    UNICODE_MODE_WINC: 'UC_WINC'
```

Comment:
Please revert this block, and re-submit it after the `develop` -> `master` merge. I try to keep the Converter based on `qmk:master`.

Reply:
Done